composer.json

```diff
@@ -15,7 +15,8 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0",
+        "php": ">=5.4",
+        "ext-intl": "*",
```

Comment:
Is it wise to release a software without specific requirements needed for full functionality?